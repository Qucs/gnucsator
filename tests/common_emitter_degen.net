# Qucs 0.0.20  /home/dow/.qucs/ch2_prj/common_emitter_degen.sch

BJT:Q2N3904_1 _net0 out _net1 out Type="npn" Is="1.4e-14" Nf="1" Nr="1" Ikf="0.025" Ikr="0" Vaf="100" Var="0" Ise="3e-13" Ne="1.5" Isc="0" Nc="2" Bf="300" Br="7.5" Rbm="0" Irb="0" Rc="2.4" Re="0" Rb="0" Cje="4.5e-12" Vje="0.75" Mje="0.33" Cjc="3.5e-12" Vjc="0.75" Mjc="0.33" Xcjc="1" Cjs="0" Vjs="0.75" Mjs="0" Fc="0.5" Tf="4e-10" Xtf="0" Vtf="0" Itf="0" Tr="2.1e-08" Temp="26.85" Kf="9e-16" Af="1" Ffe="1" Kb="0" Ab="1" Fb="1" Ptf="0" Xtb="1.5" Xti="3" Eg="1.11" Tnom="26.85" Area="1"
Vdc:V1 _net2 gnd U="20 V"
R:R1 _net0 _net2 R="110k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R3 gnd _net0 R="10k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R2 out _net2 R="10k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R4 gnd _net1 R="1k" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
Vac:V2 in gnd U=".99 V" f="5k Hz" Phase="0" Theta="0"
.TR:TR1 Type="lin" Start="0" Stop="1 ms" Points="500" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
C:C1 in _net0 C=".1 uF" V="0 V"
