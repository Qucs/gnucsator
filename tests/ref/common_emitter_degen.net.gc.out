<Qucs Dataset>
default plugins: ignored
detach_all: already installed, replacing
stashing as detach_all:0
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
Gnucap   System status
iterations: op=0, dc=0, tran=199, fourier=0, total=226
transient timesteps: accepted=27, rejected=0, total=27
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=5, subckt=0, model=1, total=6
dctran density=83.3%, ac density=83.3%

<indep time 27>
 0.
 10.E-6
 20.E-6
 36.61017E-6
 64.94516E-6
 93.28016E-6
 136.4573E-6
 179.6344E-6
 222.8116E-6
 265.9887E-6
 309.1658E-6
 352.343E-6
 395.5201E-6
 438.6972E-6
 481.8744E-6
 525.0515E-6
 568.2286E-6
 611.4058E-6
 654.5829E-6
 697.7601E-6
 740.9372E-6
 784.1143E-6
 827.2915E-6
 870.4686E-6
 913.6457E-6
 956.8229E-6
 0.001
</indep>
<dep _net0.Vt time>
 1.595271
 1.896786
 2.167633
 2.475273
 2.426266
 1.73529
 0.6469016
 0.9964477
 2.236456
 2.417161
 1.256354
 0.5848211
 1.461816
 2.505753
 2.075376
 0.847227
 0.7564942
 1.947742
 2.537396
 1.595684
 0.601139
 1.114443
 2.330213
 2.336986
 1.126494
 0.6078472
 1.599681
</dep>
<dep _net1.Vt time>
 0.9517058
 1.245192
 1.510047
 1.811786
 1.76367
 1.087771
 0.07292897
 0.3787474
 1.577479
 1.754734
 0.624835
 0.0322185
 0.8224918
 1.840026
 1.419724
 0.24163
 0.1615973
 1.294951
 1.868292
 0.9521038
 0.04175625
 0.4897068
 1.669413
 1.676054
 0.5011172
 0.04596332
 0.9559872
</dep>
<dep _net2.Vt time>
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
 20.
</dep>
<dep in.Vt time>
 0.
 0.3059268
 0.5819074
 0.9036939
 0.8828694
 0.2074499
-0.9017419
-0.5910672
 0.6502912
 0.8677129
-0.2811504
-0.9873193
-0.1388731
 0.9282402
 0.5337633
-0.7011677
-0.8320527
 0.3471974
 0.9797567
 0.06960882
-0.9501439
-0.4738172
 0.7485735
 0.7922739
-0.4115258
-0.9673444
-11.387E-12
</dep>
<dep out.Vt time>
 10.56083
 7.653191
 5.025685
 2.032778
 2.497347
 9.205893
 19.27766
 16.25335
 4.355678
 2.590284
 13.79732
 19.68739
 11.84505
 1.9195
 5.899052
 17.61629
 18.39406
 7.170867
 1.923975
 10.55053
 19.58885
 15.15027
 3.445915
 3.366151
 15.03017
 19.54778
 10.52459
</dep>
